k8s/containers/seed-update/changelogs/base/base.yml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
```

Comment:
That is a mistake in the commit - there should be no /changelogs - Again, I had to build these images locally for testing with the updated scripts